src/libraries/System.Net.WebSockets.Client/tests/ConnectTest.Http2.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Test.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.WebSockets.Client.Tests
+{
+    public class ConnectTest_Http2 : ClientWebSocketTestBase
+    {
+        public ConnectTest_Http2(ITestOutputHelper output) : base(output) { }
+
+        [Fact]
+        [SkipOnPlatform(TestPlatforms.Browser, "WebSocket over HTTP/2 is not supported on Browser")]
```

Comment:
Browsers support websockets over HTTP/2, so what is wrong with this test in the browser?

Reply:
I believe this if out of scope of this PR. As I understand it, the main goal is to get the managed implementation in for 7.0. (So I would love this to be merged before Platform Complete)

Reply:
I check the build and the reason of failure was not related to browser but to loopback server:
```
18:01:05] fail: [FAIL] System.Net.WebSockets.Client.Tests.ConnectTest_Http2.ConnectAsync_VersionNotSupported_Throws
[18:01:05] info: System.PlatformNotSupportedException : System.Net.Sockets is not supported on this platform.
[18:01:05] info:    at System.Net.Sockets.Socket..ctor(AddressFamily addressFamily, SocketType socketType, ProtocolType protocolType)
[18:01:05] info:    at System.Net.Test.Common.Http2LoopbackServer..ctor(Http2Options options)
[18:01:05] info:    at System.Net.Test.Common.Http2LoopbackServer.CreateServer(Http2Options options)
[18:01:05] info:    at System.Net.Test.Common.Http2LoopbackServer.CreateClientAndServerAsync(Func`2 clientFunc, Func`2 serverFunc, Http2Options http2Options, Int32 timeout)
[18:01:05] info:    at System.Net.WebSockets.Client.Tests.ConnectTest_Http2.ConnectAsync_VersionNotSupported_Throws()
```
From https://helix.dot.net/api/2019-06-17/jobs/ca5ef308-f1cb-491d-885d-6421f58ea3fc/workitems/WasmTestOnBrowser-System.Net.WebSockets.Client.Tests/console

There is an active issue https://github.com/dotnet/runtime/issues/69870, I will replace skip with it